static/src/javascripts/projects/common/modules/experiments/ab.spec.ts

```diff
@@ -36,11 +36,15 @@ jest.mock('common/modules/experiments/ab-ophan', () => ({
 }));
 jest.mock('lodash/memoize', () => (f: (...args: unknown[]) => unknown) => f);
 
-const cfg = window.guardian.config as {
-	page?: Record<string, unknown>;
-	switches?: Record<string, boolean>;
+// from https://github.com/joonhocho/tsdef
+type DeepPartial<T> = {
```

Comment:
Nice! I wonder if its worth moving this type out so it can be reused, seems like it could be useful in tests elsewhere.

Reply:
For the time being I think it can stay here, but I’ll share in the team chat and if someone finds more use we can move it up!